geoportal/tests/functional/test_login.py

```diff
@@ -350,6 +350,7 @@ class F:
             "functionalities": {"func": ["value"]},
         }
         self.assertEqual(login.loginuser(), expected)
+        request.response.headers["Vary"] = "Origin, Cookie"
```

Comment:
This is not an assertion, you set a value in the headers

Reply:
:-P